sdk/openai/Azure.AI.OpenAI/src/Generated/AudioTranscriptionOptions.Serialization.cs

```diff
@@ -193,6 +201,59 @@ internal static AudioTranscriptionOptions DeserializeAudioTranscriptionOptions(J
                 serializedAdditionalRawData);
         }
 
+        private BinaryData SerializeMultipart(ModelReaderWriterOptions options)
+        {
+            using MultipartFormDataRequestContent content = ToMultipartRequestContent();
+            using MemoryStream stream = new MemoryStream();
+            content.WriteTo(stream);
+            if (stream.Position > int.MaxValue)
+            {
+                return BinaryData.FromStream(stream);
+            }
+            else
+            {
+                return new BinaryData(stream.GetBuffer().AsMemory(0, (int)stream.Position));
```

Comment:
How can you be sure that the data that you want to write from the stream starts at position 0 in the stream's buffer?

Reply:
The stream only contains the mutlipart/form-data payload, when we persistent it, we will store all the data, so it start from position 0.